birch

```diff
@@ -178,10 +178,17 @@ cmd() {
             clean 
          ;;
 
-        '/next'*) 
-            chan=${cl[z = z + 1 >= ${#cl[@]} ? 0 : z + 1]} 
+        '/next'*)
+            chan=${cl[z = z + 1 >= ${#cl[@]} ? 0 : z + 1]}
+        ;;&
+
+        '/prev'*)
+            chan=${cl[z = z - 1 < 0 ? ${#cl[@]} : z - 1]}
```

Comment:
There's an off-by-one error here. Try `<= 0`?

Reply:
This works:

```
            chan=${cl[z = z - 1 < 0 ? ${#cl[@]}-1 : z - 1]}
```

Reply:
Thanks. Commited.